oak_crypto/src/hpke/mod.rs

```diff
@@ -40,15 +41,19 @@ pub(crate) type EncappedKey = <Kem as KemTrait>::EncappedKey;
 /// <https://www.rfc-editor.org/rfc/rfc9180.html#name-encryption-and-decryption>
 const MAX_SEQUENCE_NUMBER: u128 = (1 << (8 * AEAD_NONCE_SIZE_BYTES)) - 1;
 
-pub(crate) struct KeyPair {
+pub fn gen_kem_keypair() -> (PrivateKey, PublicKey) {
+    Kem::gen_keypair(&mut OsRng)
+}
+
+pub struct KeyPair {
     pub(crate) private_key: PrivateKey,
     pub(crate) public_key: PublicKey,
```

Comment:
Instead of creating a public `gen_kem_keypair` function, you could've just made these fields public.